(DOCSP-28162): Add docs release notes for w/e February 24 (#365)
mongodb/docs-app-services: docs-release-notes.md

# February 24, 2023

## Device Sync
- Configure Sync/Sync Settings: Update list of SDKs that support Asymmetric Sync to include C++ SDK
- Update many pages across Device Sync documentation for the release of the rules unification project

## Authenticate & Manage Users
- Define User Metadata: Add missing ref targets, update links to SDK pages to use bullets instead of tabs

> - Manage User Sessions: New page consisting of moved and new content. Moved get & refresh token and revoke user sessions content, new configure token expiration, verify & decode token, and end user session content.

Reviewer: ```suggestion
- Manage User Sessions: New page consisting of moved and new content. Moved get & refresh token and revoke user sessions content; new configure token expiration, verify & decode token, and end user session content.
```

## Define Data Access Permissions
- Update many pages across Rules documentation for the release of the rules unification project

## Reference

> - App Configuration Files/MongoDB Data Sources: Remove Device Sync from the info about when you would define collection-level rules in a collection's `rules.json` after release of rules unification project

Reviewer: ```suggestion
- App Configuration Files/MongoDB Data Sources: Remove Device Sync from description of when you would define collection-level rules in a collection's `rules.json` for the release of rules unification project
```
